Makefile

```diff
@@ -134,3 +147,42 @@ stop: ## stop the development server using Docker
 .PHONY: down
 down: ## Stop and remove containers, networks, images, and volumes
 	@$(COMPOSE) down
+
+###########################################
+# Translations tasks
+
+.PHONY: i18n-generate-sources
+i18n-generate-sources: ## Generate source translations file for back and front applications
+	${MAKE} generate-i18n-back;
```

Comment:
These two make targets don't seem to exist.

I have a more general comment: it would be nice to simplify these commands and choose coherent names from generic to specific. I propose:

- i18n-collect or i18n-generate
- i18n-upload
- i18n-download
- i18n-compile

I would not make specific commands for front and back because when you update translations you want to do it for the whole project and it divides the number of commands by 3... 
If you think we must, then the names could be of the form:

- i18n-upload-back
- i18n-upload-front

Reply:
I rework the makefile. I did'ny set a description message for specific tasks, they will not be shown by the help task. I would like to keep them, it's easier for me to use them when I develop.